packages/gatsby-source-contentful/src/plugin-options.js

```diff
@@ -0,0 +1,32 @@
+const defaultOptions = {
+  host: `cdn.contentful.com`,
+
+  // default is always master
+  environment: `master`,
+}
+
+const validateOptions = options => {
+  const errors = []
+  ;[`spaceId`, `accessToken`].forEach(option => {
+    if (!options[option]) {
+      errors.push(`Missing ${option}`)
+    }
+  })
+
+  return errors
+}
+
+const formatOptionsSummary = options => {
```

Comment:
Test would be nice here, just so it clarifies what you're doing here. Looking like you're spacing things out nicely and formatting the options in a table, basically?

Reply:
Also this would be a kind of cool package (that probably already exists). Basically some way to show what was passed, defaults, etc. It'd probably look nice as a table or something too!

Reply:
Yeah, I'll look into using something like https://www.npmjs.com/package/columnify and maybe implement this in reporter so it can be reused in other plugins